cmd/studyjobcontroller/v1alpha2/main.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 The Kubeflow Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ StudyJobController is a controller (operator) for StudyJob
+ StudyJobController create and watch workers and metricscollectors.
+ The workers and metricscollectors are generated from template defined ConfigMap.
+ The workers and metricscollectors are kubernetes object. The default object is a Job and CronJob.
+*/
+package main
+
+import (
+	"log"
+
+	"github.com/kubeflow/katib/pkg/api/operators/apis"
+	controller "github.com/kubeflow/katib/pkg/controller/v1alpha2"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+)
+
+func main() {
```

Comment:
Will Katib controller have hame "StudyJobController" in v1alpha2?

Reply:
updated